.travis.yml

```diff
@@ -0,0 +1,20 @@
+dist: trusty
+sudo: true
+language: ruby
+rvm:
+- 2.2
+before_install:
+- gem update --system
+- gem --version
+- gem update bundler
+- wget https://github.com/spf13/hugo/releases/download/v0.18.1/hugo_0.18.1-64bit.deb
+  && sudo dpkg -i hugo_0.18.1-64bit.deb
+- hugo version
+bundler_args: "--without=guard"
```

Comment:
We can probably expand this some, to include `pry`, `pry-byebug`, and `guard-rake`.
Really I should figure out how to make those optional installs in Bundler.